services/environments/snd/04/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+metadata:
+  name: cluster-base
+resources:
+  - kustomize.yaml
+  - ../base
```

Comment:
We may eventually want to split this out so that snd1 to snd3 have a different base from snd4 when all the services have been deployed to snd4 and the teams have switched over to using snd4.

Reply:
I'll discuss with you and then ask Lorenzo to create a story to look into splitting this out.